meltano.yml

```
@@ -35,6 +35,8 @@ plugins:
       kind: array
     - name: user_ids
       kind: array
+    - name: milestones
+      kind: object
```

Nit: expand the nested setting

```suggestion
    - name: milestones.state
      kind: options
      options:
      - label: Open
        value: open
      - label: Closed
        value: closed
      - label: all
        value: All
```